tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -0,0 +1,86 @@
+*** Settings ***
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHAdminUI.robot
+Resource        ../../Resources/ODS.robot
+Suite Setup     Set Library Search Order  SeleniumLibrary
+Suite Teardown  Set Default Access Groups Settings
+
+*** Test Cases ***
+Verify if the CRD after applying changed In admin UI
+    [Documentation]  Verify if the CRD after applying changed In admin UI
+    [Tags]  ODS-1661
+    ...     Tier1
+    ...     Sanity
+    Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear User Management Settings
+    Add OpenShift Groups To Data Science Administrators   rhods-admins  rhods-users
+    Add OpenShift Groups To Data Science User Groups    system:authenticated
+    Save Changes In User Management Setting
+    AdminGroups In OdhDashboardConfig CRD Should Be     rhods-admins  rhods-users
+    AllowedGroups In OdhDashboardConfig CRD Should Be   system:authenticated
+
```

Comment:
not sure if we want to have these 2 TCs dependent on each other in this way, even if it would make the execution faster.

Reply:
I guess we discussed the same yday to divide 1TC into 2Tc's?

Reply:
yes it is true, but I was just wondering if there was a way to make them independent. However, it is okay for now to me. We have merged test cases with a similar approach